src/mantarray_file_manager/files.py

```diff
@@ -30,7 +33,32 @@ def get_unique_files_from_directory(directory: str) -> List[str]:
 
     for path, _, files in os.walk(directory):
         for name in files:
-            unique_files.append(os.path.join(path, name))
+            if name.endswith(".h5"):
+                unique_files.append(os.path.join(path, name))
+
+    for file in unique_files:
+        well = WellFile(file)
+        index = well.get_well_index()
+        barcode = well.get_plate_barcode()
+        start_time = well.get_begin_recording()
+
+        duplicates = []
+
+        unique_files.remove(file)
+
+        for item in unique_files:
+            new_well = WellFile(item)
+            if (
+                new_well.get_well_index() == index
+                and new_well.get_plate_barcode() == barcode
+                and new_well.get_begin_recording() == start_time
+            ):
+                duplicates.append(item)
+
+        unique_files.append(file)
```

Comment:
we're going to quickly run into situations where we may not know exactly which file the user would want us to use if there are duplicates.  I think we may have discussed this on the hangout, but looks like I forgot to make it a subtask on the Asana card.  please raise an error if any duplicate files are found in the folder requested to search. include the name of both files which are duplicates of each other. the error message should tell the user that we can't proceed with analysis when duplicate files are present, so please delete whichever one they want to remove